Sarah_Alasadi/week_01/day_02/v_p.js

```diff
@@ -0,0 +1,47 @@
+
+//The Fortune Teller
+var numChild = 2;
+var nameFather = "Mohammed";
+var nameMother = "Hilah";
+var gps = "Riyadh";
+var Job = "teacher";
+
+ alert("You will be a"+" "+Job+" "+"in"+" "+gps+",and married to"+" "+nameMother+" "+"with"+" "+numChild+" "+"kids");
+
+ //The Age Calculator
+var year = 2018;
+var birthYear =1991;
+var sum = year-birthYear;
+
+alert("They are either"+" "+(sum));
+
+//The Lifetime Supply Calculator
+
+var Age = 26;
+var AgeMaximum = 90;
+var DayAmount = 50;
+var year = 12;
+var Months = 30;
+var sum = parseInt(AgeMaximum-Age);
+var E = parseFloat(year * Months);
+var A = parseInt(sum * E);
+var x = parseInt(A * DayAmount);
+alert("You will need"+" "+x+" "+"Riyal"+" "+"to last you until the ripe old age of"+" "+sum);

```

Comment:
Great work, remember that you can also add a space at the end of a sentence and that will work the same, for example:
```
var myName = "Sarah";
console.log("Hi, my name is " + myName);
```
Is the same thing as doing:
```
var myName = "Sarah";
console.log("Hi, my name is" + " " + myName);
```